Bottom: 7fef7d191425e5a3b8a550169fcb1824a1a20e44
Top:    5388462e82076ca88e044428c18e74bf17315a42
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-11-15 02:42:55 +0000

Refresh of major-re-write-to-drawing-rout

---

diff --git a/src/draw.h b/src/draw.h
index 4c6528c..94ea83d 100644
--- a/src/draw.h
+++ b/src/draw.h
@@ -82,7 +82,7 @@ void DrawRegularText (LayerTypePtr, TextTypePtr, int);
 void DrawEMark (ElementTypePtr, LocationType, LocationType, Boolean);
 void DrawHole (PinTypePtr);
 void DrawRats (BoxType *);
-void DrawSilk (int, int, BoxType *);
+void DrawSilk (int, int, const BoxType *);
 void DrawSpecialPolygon (HID * hid, hidGC DrawGC, LocationType X, LocationType Y, int Thickness, int thin_draw);
 
 /* TEMPORARY */
diff --git a/src/hid/gtk/gtkhid-main.c b/src/hid/gtk/gtkhid-main.c
index 6656139..12584da 100644
--- a/src/hid/gtk/gtkhid-main.c
+++ b/src/hid/gtk/gtkhid-main.c
@@ -614,6 +614,7 @@ ghid_set_layer (const char *name, int group, int empty)
 	    gport->trans_lines = TRUE;
 	  return PCB->RatOn;
 	}
+    }
 
   return 0;
 }
